### DISCUSSION TITLE
discord invite links expired

### DISCUSSION THREAD

**Discussion by ParthJadhav** [spacedriveapp/spacedrive#424] [Community Support] [answered]
![Screenshot 2022-10-20 at 11 22 11 PM](https://user-images.githubusercontent.com/42001064/197022178-2663f8c2-4def-46d9-9885-9f9929d886b6.png)

**Comment by brxken128** [comment]
https://discord.gg/SjsJaBhB should do it.

Note I set this to be valid for 1 day, as I don't know whether @jamiepine or someone wanted to handle this differently (like a fancy custom invite URL).

**Comment by Redbeardjunior** [comment]
Still not working...

**Comment by xPolar** [comment, marked as answer]
This invite link should work no problem!

https://discord.gg/pXjwKz7HJv
